AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -70,6 +70,11 @@
     "kubernetesRecordingRuleGroup": "KubernetesReccordingRulesRuleGroup-",
     "kubernetesRecordingRuleGroupName": "[concat(variables('kubernetesRecordingRuleGroup'), variables('clusterName'))]",
     "kubernetesRecordingRuleGroupDescription": "Kubernetes Recording Rules RuleGroup",
+    "kubernetesRecordingRuleGroupWin": "KubernetesRecordingRulesRuleGroup-Win-",
+    "kubernetesRecordingRuleGroupWin1": "KubernetesRecordingRulesRuleGroup-Win-1-",
```

Comment:
Thanks Kaveesh. Is there a name that could make more clear the difference between the two rule groups? Or is this the OSS naming convention for them?

Reply:
We created these rule groups ourselves and soham initially named them as such. Let me rename them to be more in accordance with what I see in the community.

Reply:
renamed them to : `NodeAndKubernetesRecordingRulesRuleGroup-Win-{cluster_name}` and `NodeRecordingRulesRuleGroup-Win-{cluster_name}`. Since we have a limit of 20 rules per group we have to mix both node and kubernetes recording rules in the 2nd group.

Reply:
Thanks Kaveesh!